app/repositories/DepartureRepository.scala

```diff
@@ -0,0 +1,261 @@
+/*
```

Comment:
Spikespikespike

Reply:
We need to discuss this long term. If we want to include the code for the models or get the objects via GET calls